README.md

```diff
@@ -335,7 +335,11 @@ Once you've configured an instantiated presenter the way you want, you can get t
 PostPresenter::make($post)->only('id', 'title')->get();
 ```
 
-If you wish to return all the defined values in your presenter (without any configuration) then you can use the `all` method.
+If you wish to return all the defined values in your presenter (without any configuration) then you can use the `all` method, It will ignore the values ​​defined by `only()`, `except()`, `with()`:
```

Comment:
Could you please remove the following text:

, It will ignore the values ​​defined by `only()`, `except()`, `with()`

I'm not sure that this makes the usage of `all()` much clearer that the existing text

Reply:
I was just thinking that if you don’t use `only` or `except`, the `all` method is the same as `get`, so I think can actually delete `all` method. and I occasionally get confused because Laravel Collection uses `all` return array. (Sorry! This need to open another issue to discuss)

Reply:
I will still modify it according to your instructions